compiler/luci/pass/src/helpers/LayerInfoMap.cpp

```diff
@@ -22,6 +22,134 @@
 
 namespace luci
 {
+namespace
+{
+bool is_multiple_output_node(const luci::CircleNode *node)
```

Comment:
please add empty line between { and this

Reply:
Done